api/Element.json

```diff
@@ -3321,35 +3421,116 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": "1"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": "8",
+              "notes": "No support for <code>smooth</code> behavior or <code>center</code> options."
             },
             "opera": {
-              "version_added": null
+              "version_added": "38"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": "5",
+              "notes": "No support for <code>smooth</code> behavior or <code>center</code> options."
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "5",
+              "notes": "No support for <code>smooth</code> behavior or <code>center</code> options."
             },
             "samsunginternet_android": {
               "version_added": null
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "scrollIntoViewOptions": {
+          "__compat": {
+            "description": "<code>scrollIntoViewOptions</code>",
+            "support": {
+              "webview_android": {
+                "version_added": "61",
+                "notes": "The <code>block</code> and <code>inline</code> options support the values <code>start</code>, <code>center</code>, <code>end</code>, <code>nearest</code>."
+              },
+              "chrome": {
+                "version_added": "61",
+                "notes": "The <code>block</code> and <code>inline</code> options support the values <code>start</code>, <code>center</code>, <code>end</code>, <code>nearest</code>."
+              },
+              "chrome_android": {
+                "version_added": "61",
+                "notes": "The <code>block</code> and <code>inline</code> options support the values <code>start</code>, <code>center</code>, <code>end</code>, <code>nearest</code>."
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": [
+                {
+                  "version_added": "58",
```

Comment:
This one is a bit weird because in v36 there were two notes, but in v58 one of those notes was fixed (I don't know if the other note has been fixed because the note had no linked bug). I think this is the correct way to show that?

Reply:
I think a better option would be (that’s what most notes I’ve seen do):
```json
"firefox": {
  "version_added": "36",
  "notes": [
    "No support for <code>inline</code> option.",
    "Before Firefox 58, <code>nearest</code> and <code>center</code> values for the <code>block</code> option was unsupported. See <a href='https://bugzil.la/1389274'>bug 1389274</a>."
  ]
}
```